### PR TITLE
Product inspection sample c#

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**
Product inspection app which will allow the user to scan the product and mark it as accepted/rejected. The user will scan the barcode of the product and click picture before marking it as accepted/rejected.

### CODE REVIEW

File: samples/tab-product-inspection/csharp/ProductInspection/Controllers/HomeController.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        /// </summary>
+        [HttpPost]
+        [Route("Save")]
+        public string Save()
+        {
+            var productId = Request.Form["productId"];
+            var image = Request.Form["image"];
+            var status = Request.Form["status"];
+            // Check if product id recieved is null
+            if (string.IsNullOrEmpty(productId))
```
Comment (v-chetsh) [inline samples/tab-product-inspection/csharp/ProductInspection/Controllers/HomeController.cs:85]: One line space before it.

File: samples/tab-product-inspection/csharp/ProductInspection/Manifest/manifest.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Product Inspection",
+    "full": "Product Inspection sample"
+  },
+  "description": {
+    "short": "Product Inspection",
+    "full": "This sample app demonstrate a feature where user can scan a product and mark it as approved/rejected."
```
Comment (v-chetsh) [inline samples/tab-product-inspection/csharp/ProductInspection/Manifest/manifest.json:23]: This sample app demonstrate a feature where user can scan a product, capture a image and mark it as approved/rejected.

File: samples/tab-product-inspection/csharp/ProductInspection/Manifest/manifest.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  },
+  "accentColor": "#5558AF",
+  "staticTabs": [
+    {
+      "entityId": "index",
```
Comment (v-chetsh) [inline samples/tab-product-inspection/csharp/ProductInspection/Manifest/manifest.json:28]: Why it's name is "index"??
Change it as per the tab name.

File: samples/tab-product-inspection/csharp/ProductInspection/Views/Home/Index.cshtml [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+        .flex-container {
+            display: flex;
+        }
+
+            .flex-container > div {
+                margin: 10px;
+                padding: 10px;
+            }
+
```
Comment (v-chetsh) [inline samples/tab-product-inspection/csharp/ProductInspection/Views/Home/Index.cshtml:22]: Check for proper alignment.

File: samples/tab-product-inspection/csharp/ProductInspection/Views/Home/ProductDetails.cshtml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    <script src="https://statics.teams.cdn.office.net/sdk/v1.10.0/js/MicrosoftTeams.min.js"
+            integrity="sha384-6oUzHUqESdbT3hNPDDZUa/OunUj5SoxuMXNek1Dwe6AmChzqc6EJhjVrJ93DY/Bv"
+            crossorigin="anonymous"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    <style>
+
+        #mainDiv {
+            padding-left: 5%;
+            padding-top: 2%;
```
Comment (v-chetsh) [inline samples/tab-product-inspection/csharp/ProductInspection/Views/Home/ProductDetails.cshtml:17]: Alignment is not correct.

File: samples/tab-product-inspection/csharp/ProductInspection/Views/Home/ScanProduct.cshtml [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+        function saveProduct(status) {
+            var productStatus = status ? "Approved" : "Rejected";
+            const formData = new FormData();
+            formData.append('productId', productId);
+            formData.append('image', "data:image/png;base64,"+image);
+            formData.append('status', productStatus);
+            $("#productStatus").hide();
+            $("#captureImage").hide();
+            $("#wait").show();
+            setTimeout(function(){
```
Comment (v-chetsh) [inline samples/tab-product-inspection/csharp/ProductInspection/Views/Home/ScanProduct.cshtml:101]: One line space before it.

File: samples/tab-product-inspection/csharp/ProductInspection/Views/Home/ViewDetails.cshtml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+            scanBarCode();
+        });
+        var productId;
+
+        // Method to scan barcode
+        function scanBarCode() {
+            var config = {
+                timeOutIntervalInSec: 30
+            };
+            // Method that enables the user to scan different types of barcode, and returns the product id as string.
```
Comment (v-chetsh) [inline samples/tab-product-inspection/csharp/ProductInspection/Views/Home/ViewDetails.cshtml:18]: One line space before it.

File: samples/tab-product-inspection/csharp/README.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
+﻿# Product Inspection
+
+This sample app demonstrate a feature where user can scan a product and mark it as approved/rejected.
```
Comment (v-chetsh) [inline samples/tab-product-inspection/csharp/README.md:3]: This sample app demonstrate a feature where user can scan a product, capture a image and mark it as approved/rejected.

**Review by v-chetsh — COMMENTED** [review]
Reviewed and provided my comments.

File: samples/tab-product-inspection/csharp/ProductInspection/Controllers/HomeController.cs (hunk 1 above)
Comment (v-chetsh) [reply samples/tab-product-inspection/csharp/ProductInspection/Controllers/HomeController.cs:86]: ok

File: samples/tab-product-inspection/csharp/ProductInspection/Views/Home/Index.cshtml (hunk 4 above)
Comment (v-chetsh) [reply samples/tab-product-inspection/csharp/ProductInspection/Views/Home/Index.cshtml:22]: ok

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving it.

**Review by AbhijeetThakur-msft — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**